src/main/obsRecorder.ts

```diff
@@ -133,22 +133,51 @@ const setupScene = (monitorIndex: number) => {
 
   // Correct the monitorIndex. In config we start a 1 so it's easy for users. 
   const monitorIndexFromZero = monitorIndex - 1; 
-  console.info("monitorIndexFromZero:", monitorIndexFromZero);
+  console.info("[OBS] monitorIndexFromZero:", monitorIndexFromZero);
   const { physicalWidth, physicalHeight } = displayInfo(monitorIndexFromZero);
 
+  const obsSettings = osn.NodeObs.OBS_settings_getSettings('Video').data;
+
+  // Checks if val is +-1 compare
+  const isClose = (val: number, compare: number) => {
+    return Math.abs(compare - val) <= 1;
+  };
+
+  // Checks if resolution is close enough to monitor, and if so, sets the settingString.
+  const checkRes = (resolution: string) => {
+    const [resW, resH] = resolution.split('x');
+
+    return isClose(parseInt(resW, 10), physicalWidth) && isClose(parseInt(resH, 10), physicalHeight);
+  };
+
+  // TODO: Output should eventually be moved into a setting field to be scaled down. For now it matches the monitor resolution.
```

Comment:
Where did this information come from? It's news to me - I'm quite keen to understand OBS better so if you have knowledge please share!

Reply:
![image](https://user-images.githubusercontent.com/7443232/190899887-acf5bb70-5def-4dbb-bed0-0bd9c9cebbaa.png)

Video.Base translates to Base Canvas Resolution, which is the size of the scene we are creating. For current purposes, this should always match monitor resolution.
Video.Output translates to Output Resolution, which is the video resolution we're recording at. For users where storage space is a concern, they should be able to change the output resolution to a size that they find acceptable, say 720p. Currently, with this setup, users with 4k monitors will record at 4k.

Reply:
Ah I see. That's interesting for https://github.com/aza547/wow-recorder/issues/67 if you fancy tackling another issue after!